### PR TITLE
Fix public routing

### PR DESCRIPTION
**Pull request by stvstnfrd**
Add public ELBs to a public subnet.

### CODE REVIEW

**Comment by stvstnfrd** [comment]
@kluo @caesar2164

**Comment by caesar2164** [comment]
as long as it works... 👍

File: stanford/roles/cluster/defaults/main.yml [hunk 1]
```diff
@@ -76,7 +76,7 @@ CLUSTER_SUBNETS:
       cidr: "{{ DEPLOYMENT_CIDR_BASE }}.55.0/24"
     - zone: 'us-west-1c'
       cidr: "{{ DEPLOYMENT_CIDR_BASE }}.56.0/24"
-CLUSTER_NAT_SUBNET: "{{ CLUSTER_SUBNETS['nat'][cluster_subnet_index|int] }}"
+CLUSTER_NAT_SUBNET: "{{ CLUSTER_SUBNETS['public'][cluster_subnet_index|int] }}"
```
Comment (kluo) [inline stanford/roles/cluster/defaults/main.yml]: Should `CLUSTER_NAT_SUBNET` also be renamed to `CLUSTER_PUBLIC_SUBNET`?

Comment (stvstnfrd) [reply stanford/roles/cluster/defaults/main.yml]: Ah, probably.

Comment (stvstnfrd) [reply stanford/roles/cluster/defaults/main.yml]: Actually, no.
Where it's used in the play, it's explicitly for creating the NAT box/subnet, so it's more just saying "the nat subnet should use the public subnet"

**Comment by kluo** [comment]
+1

**Comment by stvstnfrd** [comment]
I confused github with a rebase; this is merged.

**Pull request closed without merging**